[2025-02-05T06:40:36.820+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:40:36.868+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_group_example.end_task scheduled__2024-02-22T00:00:00+00:00 [queued]>
[2025-02-05T06:40:36.871+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_group_example.end_task scheduled__2024-02-22T00:00:00+00:00 [queued]>
[2025-02-05T06:40:36.871+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-05T06:40:36.875+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): end_task> on 2024-02-22 00:00:00+00:00
[2025-02-05T06:40:36.877+0000] {standard_task_runner.py:57} INFO - Started process 2585 to run task
[2025-02-05T06:40:36.879+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'task_group_example', 'end_task', 'scheduled__2024-02-22T00:00:00+00:00', '--job-id', '280', '--raw', '--subdir', 'DAGS_FOLDER/task_group_dag.py', '--cfg-path', '/tmp/tmpnn0gyy8_']
[2025-02-05T06:40:36.880+0000] {standard_task_runner.py:85} INFO - Job 280: Subtask end_task
[2025-02-05T06:40:36.885+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:40:36.897+0000] {task_command.py:415} INFO - Running <TaskInstance: task_group_example.end_task scheduled__2024-02-22T00:00:00+00:00 [running]> on host 3849bfb82152
[2025-02-05T06:40:36.901+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:40:36.923+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_group_example' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-22T00:00:00+00:00'
[2025-02-05T06:40:36.924+0000] {logging_mixin.py:151} INFO - 执行任务: 结束任务
[2025-02-05T06:40:36.924+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-05T06:40:36.927+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=task_group_example, task_id=end_task, execution_date=20240222T000000, start_date=20250205T064036, end_date=20250205T064036
[2025-02-05T06:40:36.938+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-05T06:40:36.944+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
